### ISSUE TITLE
[1.16.5] Fullscreen closes with `Fullscreen.render not running` in console

### ISSUE THREAD

**Issue by Damonenwolf** [TeamJM/journeymap#436]
Modpack:   Monumental Experience v. 17.0.22

#### Forge Version:     1.16.5    v.36.2.35

1.   I open the Fullscreen map.
2.   when I hover the mouse over the map, the map closes , chunks are revealed and loaded

[21:51:52] [main/WARN] [StatTimer] Fullscreen.render is not running.
[21:51:54] [main/ERROR] [Fullscreen] Unexpected exception in jm.fullscreen.render(): java.lang.NullPointerException

[journeymap.log](https://github.com/TeamJM/journeymap/files/8682178/journeymap.log)
[crash-2022-05-12_21.54.19-client.txt](https://github.com/TeamJM/journeymap/files/8682187/crash-2022-05-12_21.54.19-client.txt)

**Comment by NightTTQ** [comment]
I also encountered the same situation.

JM Version: 5.8.3
Forge Version: 1.16.5 36.2.35

```
[14:46:18] [Render thread/ERROR]: Unexpected exception in jm.fullscreen.render(): java.lang.NullPointerException: Cannot invoke "String.length()" because "☃" is null
	at net.minecraft.util.text.TextProcessing.func_238340_a_(SourceFile:88)
	at net.minecraft.util.text.TextProcessing.func_238339_a_(SourceFile:84)
	at net.minecraft.util.text.TextProcessing.func_238337_a(SourceFile:128)
	at net.minecraft.util.text.ITextComponent.func_230534_b_(SourceFile:112)
	at net.minecraft.util.text.ITextComponent.func_230439_a_(SourceFile:79)
	at net.minecraft.util.text.TextProcessing.func_238343_a_(SourceFile:128)
	at net.minecraft.util.text.CharacterManager.func_238356_a_(SourceFile:47)
	at net.minecraft.client.gui.FontRenderer.func_238414_a_(SourceFile:329)
	at journeymap.client.render.draw.DrawUtil.drawLabel(DrawUtil.java:236)
	at journeymap.client.render.draw.DrawUtil.drawLabel(DrawUtil.java:210)
	at journeymap.client.ui.fullscreen.layer.BlockInfoLayer$BlockInfoStep.draw(BlockInfoLayer.java:330)
	at journeymap.client.render.map.GridRenderer.draw(GridRenderer.java:463)
	at journeymap.client.render.map.GridRenderer.draw(GridRenderer.java:439)
	at journeymap.client.ui.fullscreen.Fullscreen.drawMap(Fullscreen.java:1525)
	at journeymap.client.ui.fullscreen.Fullscreen.func_230430_a_(Fullscreen.java:441)
	at net.minecraftforge.client.ForgeHooksClient.drawScreenInternal(ForgeHooksClient.java:363)
	at net.minecraftforge.client.ForgeHooksClient.drawScreen(ForgeHooksClient.java:356)
	at net.minecraft.client.renderer.GameRenderer.func_195458_a(GameRenderer.java:492)
	at net.minecraft.client.Minecraft.func_195542_b(Minecraft.java:977)
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:607)
	at net.minecraft.client.main.Main.main(Main.java:184)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at net.minecraftforge.fml.loading.FMLClientLaunchProvider.lambda$launchService$0(FMLClientLaunchProvider.java:37)
	at cpw.mods.modlauncher.LaunchServiceHandlerDecorator.launch(LaunchServiceHandlerDecorator.java:37)
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:54)
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:72)
	at cpw.mods.modlauncher.Launcher.run(Launcher.java:82)
	at cpw.mods.modlauncher.Launcher.main(Launcher.java:66)

[14:46:18] [Render thread/WARN]: Fullscreen.render is not running.
```

**Comment by mysticdrew** [comment]
> I also encountered the same situation.
> 
> JM Version: 5.8.3 Forge Version: 1.16.5 36.2.35
> 
@NightTTQ 
upload your full log file, DO NOT copy and paste it in the issue

**Comment by mysticdrew** [comment]
I am unable to replicate this issue with any consistency, I have noticed it once but I have been unable to get it to happen again. If either of you are able to replicate it consistently please come into our discord and I can give you a test version which I "think" I have resolved but without being able to replicate it, I am not 100% sure

**Comment by tgstyle** [comment]
I am seeing the same issue on the same versions. In my case as long as you mouse over explored areas the map stays open. If you mouse over an unexplored area the map closes.

**Comment by mysticdrew** [comment]
> I am seeing the same issue on the same versions. In my case as long as you mouse over explored areas the map stays open. If you mouse over an unexplored area the map closes.

see above.. I need someone that can replicate to test. 
I am locking this now as commenting "I have the same issue" is no help any longer. I know it is an issue and I have a potential fix but I need it tested so I can release it.

**Comment by mysticdrew** [comment]
This is fixed as of 5.8.4. It will be available on curseforge shortly.. as soon as it is approved.
